src/base/bittorrent/session.cpp

```diff
@@ -1078,7 +1090,7 @@ void Session::initializeNativeSession()
     pack.set_str(lt::settings_pack::user_agent, USER_AGENT);
     pack.set_bool(lt::settings_pack::use_dht_as_fallback, false);
     // Speed up exit
-    pack.set_int(lt::settings_pack::stop_tracker_timeout, 1);
+    pack.set_int(lt::settings_pack::stop_tracker_timeout, stop_tracker_timeout());
```

Comment:
Since this will not be a hardcoded setting anymore, I think it would make more sense to initialize it at
`Session::loadLTSettings` instead.

Something like:

`settingsPack.set_int(lt::settings_pack::stop_tracker_timeout, stopTrackerTimeout());`